packages/phone-number-privacy/signer/src/server.ts

```diff
@@ -52,14 +53,28 @@ export function createServer() {
   }
 }
 
-async function callAndMeasureLatency(
+async function callWithTimeout(
```

Comment:
I think we can set this at the Express level in index.ts
```
server.listen(port, () => {
    logger.info(`Server is listening on port ${port}`)
  }).setTimeout(1000)
  ```
  https://nodejs.org/dist/latest-v6.x/docs/api/http.html#http_server_settimeout_msecs_callback

Reply:
+1

Though it would be nice in either approach to add a prometheus metric to track how many requests time out

Reply:
Yea my concern is that if we do it at the express level the latency measurement will never wrap up and we'll be left without insight into how many timeouts we're seeing

Reply:
What we'll see with this implementation when signers timeout is signers reporting latency measurements that hit the ceiling set by the timeout parameter. We'll also see a prometheus counter tracking the number of timeouts.

Reply:
I think we should add this at the server level too as a last resort and set a higher limit

Reply:
👍